tactics/declare.ml

```diff
@@ -11,20 +11,23 @@
 (** This module is about the low-level declaration of logical objects *)
 
 open Pp
-open CErrors
 open Util
 open Names
-open Libnames
-open Globnames
-open Constr
 open Declarations
 open Entries
+open Safe_typing
 open Libobject
 open Lib
-open Impargs
-open Safe_typing
-open Cooking
-open Decl_kinds
+
+(* This was meant as error for schemes, not clear how it works *)
+exception SchemeAlreadyDeclared of Pp.t
```

Comment:
>This was meant as error for schemes

Isn't it the error for when you do 2 definitions with the same name? It doesn't seem to have much to do with schemes.

Reply:
Indeed, I've amended that; I wonder why I did that change, I was digging in history and likely got lost.